src/main/java/org/folio/rest/impl/InitAPIs.java

```diff
@@ -44,6 +46,9 @@ public void init(Vertx vertx, Context context, Handler<AsyncResult<Boolean>> res
       new ServiceBinder(vertx)
         .setAddress(EVENT_CONFIG_PROXY_CONFIG_ADDRESS)
         .register(ConfigurationService.class, ConfigurationService.create(vertx));
+      new ServiceBinder(vertx)
```

Comment:
just for my information, why are these needed?

I was expecting to just make an api call to mod-users.

Reply:
The user service is not necessary, but it was part of the AC and it doesn't hurt to encapsulate mod-users calls into a service. I am not sure about the specific kind of service used it mod-login, I just tried to keep the new one consistent with the others.